iml-services/iml-ntp/src/main.rs

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+#![feature(async_await)]
+
+mod error;
```

Comment:
probably better to have a lib.rs with this in it.

Reply:
sure, sounds good.

Reply:
Created a lib.rs and define the error mod as well as the iml_ntp mod. To pull the error in I just use `use crate::error`.